config/jest/server/start.mjs

```diff
@@ -1,42 +1,10 @@
-import { getServers, setup } from 'jest-dev-server'
-import waitOn from 'wait-on'
+import { setup } from 'jest-dev-server'
 
-import config from '../../index.js'
-
-import serverStop from './stop.mjs'
-
-const PORT = process.env.PORT || config.ports.test
+import config from '../../../jest-dev-server.config.js'
 
 /**
  * Start web server
  */
 export default async function serverStart () {
-  const servers = await getServers()
-
-  // Server start timeout
-  let timeout = 1000
-
-  // Server stopping?
-  if (servers.some(({ signalCode }) => signalCode === 'SIGTERM')) {
-    await serverStop() // Wait for server to stop
-    timeout = 0 // No need to wait for start
-  }
-
-  // Wait until ready check
-  const ready = ({ timeout = 30000 } = {}) => waitOn({
-    resources: [`tcp:${PORT}`],
-    timeout
-  })
-
-  // Wait until ready (or start up)
-  try {
-    await ready({ timeout })
-  } catch (error) {
-    await setup({
-      // Ensure PORT works (on Windows) + SIGINT/SIGTERM signal events
-      command: `cross-env-shell PORT=${PORT} node app/start.js`,
-      port: PORT
-    })
-    await ready()
-  }
+  await setup(config)
```

Comment:
**suggestion** Similar suggestion to `browserClose`, we don't need to `await` and can just return the `Promise` from `setup` 😄

Reply:
Ahh so all these Jest hooks were previously typed with `Promise<void>`

We're not adding JSDoc types or extending anything anymore so shall we simplify even more?

```suggestion
  export default () => setup(config)
```

Reply:
Pushed up Jest hooks using the shorthand above 👍